ursabot/docker.py

```diff
@@ -224,11 +224,10 @@ def apt(*packages):
     """Generates apt install command"""
     template = dedent("""
         apt update -y -q && \\
-        apt install -y -q \\
-        {} && \\
+        apt install -y -q {} && \\
```

Comment:
@kszucs I'm not sure why I needed this to make it work.

Reply:
These are just making the dockerfiles readable - shouldn't affect the image building.